packages/cascara/src/ui/ChatMessages/themes/barista/components/Button/buttonStyles.js

```diff
@@ -1,4 +1,4 @@
-import * as _ from 'lodash';
+import { isNil } from 'lodash';
```

Comment:
I am going to create us a separate task to remove `lodash` completely as a dependency so we can keep our code more functional in style. Not in scope for this, but just a heads up that we will be removing it from Cascara. I think this was code for theming copied over directly from FluentUI themes. This is probably the place we are using it most... and also why there were direct imports of `* as _` going on.

No need to change these now as this will introduce further risk and should be done separately. Mostly a heads up. Go ahead and mark this conversation as resolved once you have seen it.